doc/dynamic-port-breakout/sonic-dynamic-port-breakout-HLD.md

```diff
@@ -1190,6 +1192,73 @@ All thee attribute that coud be changed in orchagent should be able to be brough
 # Warm reboot support
 Syncd changes are required as mentioned above. The PR need to be merged and tested.
 
+# Port breakout workflow additions 
+xcvrd (transceiver daemon) running as part of PMON docker container, detects optical module (transceiver) presence.
+If transceiver is found as QSFP-DD, it initiates and orchestrates entire CMIS FSM until module ready state.
+As part of enabling CMIS FSM with port breakout, found out that port breakout feature is not supported for QSFP-DD optical modules.
+
+**Few key things to take into account prior to getting into workFlow**
+
+A NxS breakout cable inserted implies following
+- A physical port is broken down into N sub-ports (logical ports)
+  - sub-ports are numbered as 8/N i.e.
+  - For 4x100G breakout optical module inserted in physcial Ethernet1 port, implies: Ethernet8, Ethernet10, Ethernet12, Ethernet14
+  - Note: This is done is this manner to keep such assignments uniform across various breakout modes viz.1x, 2x, 4x, 8x
+- Speed of each sub-port is S Gpbs
+- Unique subport# is assigned to each of the N sub-port starting with subport# 1 (and sequentially incrementing with each sub-port)
+
+**Following is the new design (workflow) for 'port breakout' to work end-to-end with 'CMIS enabled' (in SONiC):**
```

Comment:
Not sure what is the intention with single quotes for 'port breakout' and 'CMIS enabled'. Did you mean using ``?

Reply:
Intent is to highlight right keywords with right combination